tests/kolainst/destructive/apply-live

```diff
@@ -141,6 +148,23 @@ assert_not_file_has_content_literal status.txt 'LiveDiff:'
 
 echo "ok livefs reset"
 
+# testing apply-live when changes made to systemd unit while upgrade
+rpm-ostree cleanup -pr
+rpm-ostree install pkgsystemd
+/tmp/autopkgtest-reboot "2"
+;;
+"2")
+pkgsystemd > pkg_version.txt
+assert_file_has_content_literal pkg_version.txt '1.0-1'
+rm -f pkg_version.txt
+sed -i -e 's,rpm-repos/0,rpm-repos/1,' /etc/yum.repos.d/libtest.repo
+rpm-ostree upgrade
+rpm-ostree apply-live --allow-replacement | tee out_ap.txt
+pkgsystemd > pkg_version.txt
+assert_file_has_content_literal pkg_version.txt '2.0-1'
+rm -f pkg_version.txt
+assert_file_has_content out_ap.txt 'Successfully updated running filesystem tree; Following services may need to be restarted: /pkgsystemd.service'
```

Comment:
The leading `/` here looks strange.  I suspect it it's also going to look better if we print each service on its own line, rather than joining them into a single line.